jvm/src/main/kotlin/app/cash/trifle/internal/validators/X509CertChainValidator.kt

```diff
@@ -41,6 +47,8 @@ internal class X509CertChainValidator(certAnchor: Certificate) : CertChainValida
     return try {
       PATH_VALIDATOR.validate(X509FACTORY.generateCertPath(x509Certs), pkixParams)
       true
+    } catch (e: CertPathValidatorException) {
```

Comment:
just trying to understand the conditions of re-throwing exceptions - why are we only doing so for this particular checked exception

Reply:
I don't have a strong reason for this.  I think I added a carve out from the previous behavior because that was the targeted change I wanted, but I see no reason not to throw other exceptions too.